### ISSUE TITLE
centrifuge-kreport values do not ad 

### ISSUE THREAD

**Issue by pablorr24** [DaehwanKimLab/centrifuge#276]
Hi,
I've been working on Centrifuge, and using the centrifuge-kreport tool to convert my results to kraken2 format. I'm using:

centrifuge-kreport -x /.../database/db_prefix centrif_report.tsv > kreport.txt

I am getting a kraken report but the values seem to differ. In the screenshot the Centrifuge report (above) values for Bacillus differ from those on the kraken report (below). I'm not sure if I'm interpreting something incorrectly, or there might be an issue with the code. 

![image](https://github.com/DaehwanKimLab/centrifuge/assets/92135285/0232cb27-92df-48fd-9817-b0e0682ba1ac)

Best regards,
Pablo R

**Comment by mourisl** [comment]
Indeed, the 5 reads marked in the numUniqueReads should be reflected in the report. I'll check whether I can reproduce this issue on our data or ERR3077553.

**Comment by pablorr24** [comment]
Hi, do you have any updates on this?

**Comment by mourisl** [comment]
I think one reason could be due to the taxonomy tree structure. When there are too many multiple mapping, Centrifuge will try to reduce the reported taxonomy IDs by promoting to higher taxonomy levels. In the main program, the promote will be to the standard taxonomy levels, like species, genus, family,.... In the centrifuge-kreport, the promotion may be to some non-standard taxonomy levels, like subgenus. I guess in your case, the other 4 reads may be uniquely promote to ranks like superspecies, subgenus, so the number is a bit off.
